src/flamegpu/util/detail/compute_capability.cu

```diff
@@ -90,7 +90,7 @@ std::vector<int> compute_capability::getNVRTCSupportedComputeCapabilties() {
 
 int compute_capability::selectAppropraiteComputeCapability(const int target, const std::vector<int>& architectures) {
     int maxArch = 0;
-    for (const int &arch : architectures) {
+    for (const int arch : architectures) {
```

Comment:
```suggestion
    for (const int &arch : architectures) {
```

Removing the reference here causes the for loop to make copies.

Reply:
I think a copy in this case is also fine as it is a primitive type. Will commit the suggestion anyway.